### PR TITLE
Migrate Okta to YAML-only

### PR DESCRIPTION
**Pull request by alxarch**
### Background

Okta logs can be ported to YAML.

### Changes

* Adds DetailEntry field
* Fixes AuthenticationProvider field
* Adds samples and tests

### Testing

* pantherlog test

### CODE REVIEW

File: schemas/logs/okta/tests/samples/generate_tests.jq [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Fix timestamp trailing zeros to match Go formatting
```
Comment (kostaspap) [inline schemas/logs/okta/tests/samples/generate_tests.jq:1]: Can you add a description about how this is meant to be used?

File: schemas/logs/okta/tests/generated_tests.yml [hunk 2]
```diff
@@ -0,0 +1,506 @@
+# Copyright (C) 2020 Panther Labs Inc
+#
+# Panther Enterprise is licensed under the terms of a commercial license available from
+# Panther Labs Inc ("Panther Commercial License") by contacting contact@runpanther.com.
+# All use, distribution, and/or modification of this software, whether commercial or non-commercial,
+# falls under the Panther Commercial License to the extent it is permitted.
+
+name: Okta.Systemlog.Sample <stdin> 1
```
Comment (georgepsarakis) [inline schemas/logs/okta/tests/generated_tests.yml:8]: Really minor: `<stdin>` ?

File: schemas/logs/okta/tests/samples/systemlog_samples.jsonl [hunk 3]
```diff
@@ -0,0 +1,100 @@
...rowser":"CHROME"},"zone":"null","device":"Computer","id":null,"ipAddress":"1.1.1.1","geographicalContext":{"city":"Madrid","state":"Madrid","country":"Spain","postalCode":"28005","geolocation":{"lat":40.4143,"lon":-3.7016}}},"authenticationContext":{"authenticationProvider":null,"credentialProvider":null,"credentialType":null,"issuer":null,"interface":null,"authenticationStep":0,"externalSessionId":"102iCf9Im9wSEmaVMYZNbEOMA"},"published":"2020-05-01T10:20:22.243Z","version":"0","request":{"ipChain":[{"source":null,"ip":"1.1.1.1","geographicalContext":{"city":"Madrid","state":"Madrid","country":"Spain","postalCode":"28005","geolocation":{"lon":-3.7016,"lat":40.4143}},"version":"V4"}]},"displayMessage":"User login to Okta","eventType":"user.session.start","debugContext":{"debugData":{"deviceFingerprint":"d46b17d2d7119ece5a466d6e6d4285db","requestId":"Xqv35Wg7bT74OrlHVCM7JgAAFHM","requestUri":"/api/v1/authn","threatSuspected":"false","url":"/api/v1/authn?"}},"legacyEventType":"core.user_auth.login_success","securityContext":{"asNumber":null,"asOrg":null,"isp":null,"domain":null,"isProxy":null},"uuid":"5d730bf4-8b95-11ea-8cbe-bf2eaab2e9ef","outcome":{"result":"SUCCESS","reason":null}}
```
Comment (georgepsarakis) [inline schemas/logs/okta/tests/samples/systemlog_samples.jsonl:1]: Do we need this file now that all samples have been set as test inputs and transferred to `generated_tests.yml` ?

**Review by georgepsarakis — COMMENTED** [review]
Nice work!

File: schemas/logs/okta/tests/samples/systemlog_samples.jsonl (hunk 3 above)
Comment (alxarch) [reply schemas/logs/okta/tests/samples/systemlog_samples.jsonl:1]: I want to stuff samples in the repo as well. We will have sample-testing soon (shouldparse PR) and these would be useful to test for conflicts and for E2E testing.

File: schemas/logs/okta/tests/generated_tests.yml (hunk 2 above)
Comment (alxarch) [reply schemas/logs/okta/tests/generated_tests.yml]: Fixed

File: schemas/logs/okta/tests/samples/systemlog_samples.jsonl (hunk 3 above)
Comment (alxarch) [reply schemas/logs/okta/tests/samples/systemlog_samples.jsonl:1]: There `samples` dir was duplicated by mistake, this was fixed.

**Review by kostaspap — APPROVED** [review]

**Pull request merged**